Anytype/Sources/ServiceLayer/ObjectHeaderUploadingService/ObjectHeaderUploadingService.swift

```diff
@@ -0,0 +1,112 @@
+import UIKit
+import Services
+import AnytypeCore
+import Combine
+
+protocol ObjectHeaderUploadingServiceProtocol: AnyObject {
+    
+    func coverUploadPublisher(objectId: String, spaceId: String) -> AnyPublisher<ObjectHeaderUpdate, Never>
```

Comment:
Header handle uploading state from this method.
Old implementation used `actionHandler: @escaping (ObjectSettingsAction) -> Void`